Issue a deprecation warning if a projects directory doesn't exist or can't be written to (#26509)
gradle/gradle: subprojects/docs/src/docs/userguide/releases/upgrading/upgrading_version_8.adoc

If you are using this task *without applying `grooy-base`*, you will have to manually set compatibility versions on these tasks.
In general, the `groovy-base` plugin should be applied whenever working with Groovy language tasks.

[[deprecated_missing_project_directory]]
==== Including projects without an existing directory

Gradle will warn if a project is added to the build where the associated `projectDir` does not exist or is not writable.

> Starting with version 9.0 Gradle will refuse to run builds where any project directory is missing or read-only.

Reviewer: ```suggestion
Starting with version 9.0, Gradle will not run builds if a project directory is missing or read-only.
```
